spec/spec_helper.rb

```diff
@@ -13,6 +15,16 @@
 rescue LoadError
 end
 
+def yaml_loader(path, spec_name=nil)
+  file =  File.expand_path(path)
+  raise "File #{file} cannot be found!" unless File.exist?(file)
+  raw_config = ERB.new(File.read(file)).result
+  raise "No configuration text found!" if raw_config.empty?
+  data = YAML.load(raw_config)
+  raise "No configuration data found!" if data.empty?
+  spec_name ? data[spec_name] : data
+end
```

Comment:
I'm okay with your first suggested change.

I don't agree with another PR because -- this change is necessary in order to run the specs and apply Procore-style `.env` envar management and usage in the `database.yml` file.